signalfx-org-metrics/metrics.yaml

```diff
@@ -1,143 +1,267 @@
-    SignalFx drops spans it receives after the ingestion volume limit is reached.
+sf.org.apm.numAddSpansCalls:
+  brief: The number of calls to the `/v2/trace` endpoint.
+  description: 'The number of calls to the `/v2/trace` endpoint.
 
-    Dimension(s): `orgId`
+  Dimension(s): `orgId`
 
-    Data resolution: 1 second'
+  Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.numSpansDroppedThrottle
+  title: sf.org.apm.numAddSpansCallsByToken
 
-sf.org.apm.numSpansDroppedThrottleByToken:
-  brief: The number of spans SignalFx accepts for a specific access token
-    beyond the allowed ingest volume.
-  description: 'The number of spans SignalFx accepts for a specific access token
-    beyond the allowed ingest volume. SignalFx drops spans it receives after
-    the ingestion volume limit is reached.
+sf.org.apm.numAddSpansCallsByToken:
+  brief: The number of calls to the `/v2/trace` endpoint for a specific access token.
+  description: 'The number of calls to the `/v2/trace` endpoint for a specific access token.
 
-    Dimension(s): `orgId`, `tokenId`
+  Dimension(s): `orgId`, `tokenId`
 
-    Data resolution: 1 second'
+  Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.numSpansDroppedThrottleByToken
+  title: sf.org.apm.numAddSpansCallsByToken
 
-sf.org.apm.grossSpansReceived:
-  brief: The number of spans SignalFx receives before filtering and throttling.
-  description: 'The number of spans SignalFx receives before filtering or throttling.
+sf.org.apm.numContainers:
+  brief: The number of containers actively sending data to Splunk APM.
+  description: 'The number of containers actively sending data to Splunk APM.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 minute'
+  metric_type: gauge
+  title: sf.org.apm.numContainers
+
+sf.org.apm.numHosts:
+  brief: The number of hosts that are actively sending data to Splunk APM.
+  description: 'The number of hosts that are actively sending data to Splunk APM.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 minute'
+  metric_type: gauge
+  title: sf.org.apm.numHosts
+
+sf.org.apm.grossSpanBytesReceived:
+  brief: The number of bytes Splunk APM receives from spans after decompression
+    but before filtering and throttling.
+  description: 'The number of bytes Splunk APM receives from spans after
+    decompression but before filtering and throttling.
 
   Dimension(s): `orgId`
 
   Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.grossSpansReceived
+  title: sf.org.apm.grossSpanBytesReceived
 
-sf.org.apm.grossSpansReceivedByToken:
-  brief: The number of spans SignalFx receives for a specific access token
-    before filtering and throttling.
-  description: 'The number of spans SignalFx receives for a specific access token
-    before filtering or throttling.
+sf.org.apm.numSpanBytesReceived:
+  brief: The number of bytes Splunk APM accepts for a span after decompression
```

Comment:
`for a span` --> `from all ingested span data`
Not for _a_ span, but bytes of _all_ spans ingested